platform/nvidia-bluefield/recipes/gh-helpers.mk

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+BF_ASSETS_GITHUB_REPO = nvidia-sonic/sonic-bluefield-packages
+SDK_ASSETS_GITHUB_TOKEN = 
```

Comment:
How do we plan to handle this, shouldn't we be hosting these packages in a public repo instead?

Reply:
Yes, I'm working on moving releases to the public repository.